platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -374,101 +369,91 @@ sep24:
   # the request may contain KYC fields. If `kyc_fields_forwarding` is enabled, the SEP server will call the
   # `PUT /customer` endpoint of the business server to create or update the KYC fields.
   kyc_fields_forwarding:
+    # Whether to enable the KYC fields forwarding.
     enabled: true
 
+######################
+# SEP-31 Configuration
+######################
 sep31:
-  ## @param: enabled
-  ## @type: bool
-  ## Whether to enable SEP-31 for this instance of AP server.
+  ## Whether to enable SEP-31
   #
   enabled: false
-  ## @param: payment_type
-  ## @default: STRICT_SEND
-  ## Used to determine how amount_in is calculated from amount in the POST /transaction call
-  ## @supported_values:
-  ##     STRICT_SEND: amount_in = amount
-  ##     STRICT_RECEIVE: amount_in = amount + fee
+  # The `payment_type` must be one of the following:
+  #     STRICT_SEND: amount_in = amount
+  #     STRICT_RECEIVE: amount_in = amount + fee
   payment_type: STRICT_SEND
 
-  ## @param: deposit_info_generator_type
-  ## @default: self
-  ## Used to choose how the SEP-31 deposit information will be generated, which includes the
-  ##     deposit address, memo and memo type.
-  ## @supported_values:
-  ##     self: the memo and memo type are generated in the local code, and the distribution account is used for the deposit address.
-  ##     circle: the memo and memo type are generated through Circle API, as well as the deposit address.
-  ##     api: the memo and memo type are generated through calling the anchor's GET /unique_address endpoint.
+  # Used to choose how the SEP-31 deposit information will be generated, which includes the
+  #     deposit address, memo and memo type.
+  # The `deposit_info_generator_type` must be one of the following:
+  #     self: the memo and memo type are generated in the local code, and the distribution account is used for the deposit address.
+  #     circle: the memo and memo type are generated through Circle API, as well as the deposit address.
+  #     api: the memo and memo type are generated through calling the anchor's GET /unique_address endpoint.
   #
   deposit_info_generator_type: self
 
+######################
+# SEP-38 Configuration
+######################
 sep38:
-  ## @param: enabled
-  ## @type: bool
-  ## Whether to enable SEP-38 for this instance of AP server.
+  # Whether to enable SEP-38
   #
   enabled: false
 
 
 ##########################
-## Metrics Configuration
+# Metrics Configuration
 ##########################
-## The metrics are exposed at the port specified by sep_server.management_server_port
+# The metrics are exposed at the port specified by each server's (eg: sep, platform, event processor and observer)
+# management_server_port
 metrics:
-  ## @param: enabled
-  ## @type: bool
-  ## If true, enable metrics endpoints for third-party services (ex: prometheus) to scrape
-  #
+  # If true, enable metrics will be enabled.
   enabled: false
```

Comment:
If it's not enabled, there is nothing running on `management_server_port` right?

Reply:
According to what we have today [here](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/platform/src/main/java/org/stellar/anchor/platform/configurator/SepConfigManager.java#L44), that's correct.